fixtures/issue_182.py

```diff
@@ -0,0 +1 @@
+_POS_INF = 1e10000
```

Comment:
nit: Please include a newline at EOF

Reply:
I would rather leave it as-is. It's not exactly related to the test case, but the driver should be able to handle missing EOFs, specifically when remapping positions.

Reply:
Having a test for this case makes sense, but I'd rather we do that intentionally in one place, so that we don't get so many spurious changes on updates (editors often add the missing newline later). 

In fact I'm pretty sure we already have several _other_ examples of this. 😄